src/main/java/de/retest/web/util/TextAttributeUtil.java

```diff
@@ -0,0 +1,22 @@
+package de.retest.web.util;
+
+import static de.retest.web.AttributesConfig.TEXT;
+
+import de.retest.recheck.ui.descriptors.TextAttribute;
+
+public class TextAttributeUtil {
+
+	private TextAttributeUtil() {}
+
+	public static final String PRE_ELEMENT = "pre";
```

Comment:
Does it make sense to use someting like `/pre`, `pre/`, or `/pre` so we don't accidentally apply the same logic when something match `*pre*`?